e,f,j,l,o,s,w updates for the University of Southern California

KeganStrawn opened the pull request:
# Contributing to CSrankings

Thanks for contributing to CSrankings! Please read and indicate you agree with **all** these guidelines to getting your pull request accepted.

**If you find CSrankings useful, [please consider becoming a sponsor](https://github.com/sponsors/csrankings).**

**NOTE:** IF YOU DO NOT FOLLOW THE STEPS BELOW, YOUR COMMIT MAY BE SUMMARILY REJECTED.

1. _Use a reasonable title that explains what the PR corresponds to (as in, not "Update csrankings-x.csv").

1. _Do not modify any files except `csrankings-[a-z].csv` or (if needed) `country-info.csv`._

1. _Do not use Excel to edit any .csv files; Excel incorrectly tries to
convert some Google Scholar entries to formulas, corrupting the
database. Use a text editor like emacs or NotePad instead._

1. _Insert new faculty **in alphabetical order** (not at the end) in the appropriate `csrankings-[a-z].csv` files._ **Do not modify `csrankings.csv`, which is auto-generated.**

1. _Check to make sure that you have no spaces after commas, or any missing fields._

1. _Check to make sure the home page is correct._

1. _Make sure the Google Scholar IDs are just the alphanumeric identifier (not a URL or with `&hl=en`)._

1. Check to make sure the name corresponds to the DBLP entry (look it up at http://dblp.org).

1. If a faculty member is not in a CS department or similar, include a comment explaining how they meet the inclusion criteria (see below).

1. _Read and check **all** the boxes below by filling them in with an X._

**Inclusion criteria**

- [ ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track research
faculty members on a given campus who can *solely* advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department. Faculty should also have a 75%+ time appointment (check
`old/industry.csv` for faculty who are now more than 25% in industry).

**Updating an affiliation or home page**

- [ ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings-[a-z].csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put `NOSCHOLARPAGE`.

**Adding one or more faculty members (including an entire department)**

- [ ] If the department is not yet listed in CSrankings, the entire CS faculty needs to be added (not just one faculty member).

- [ ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings-[a-z].csv` (**the letters correspond to the first letter of the faculty members' names**); include disambiguation suffixes like `0001` as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, *all of them need to be listed*. Do not update `dblp-aliases.csv`.

- [ ] If the institution you are adding is not in the US,
update `country-info.csv` and add *all* of the faculty in the CS department.

**(Advanced) Quick contribution via a shallow clone** 

We recommend that you use the GitHub web user interface to make changes. However, it may be more convenient
to clone the repository for larger-scale changes.

However, a full clone of the CSrankings repository is almost 2GB.  To
contribute a change without creating a full local clone of the
CSrankings repo, you can perform a _shallow clone_.  To do so, follow these
steps:

1. Fork the CSrankings repo.  If you have an existing fork, but it is not up to date with the main repository, this technique may not work.  If necessary, delete and re-create your fork to get it up to date.  (Do **not** delete your existing fork if it has unmerged changes you want to preserve!)

1. Do a shallow clone of your fork: `git clone --depth 1 https://github.com/yourusername/CSrankings`.  This will only download the most recent commit, not the full git history.

1. Make your changes on a branch, push them to your clone, and create a pull request on GitHub as usual.

If you want to make another contribution and some time has passed, perform these steps again, creating a fresh fork and shallow clone.

KeganStrawn committed:
e,j,l,o,s,w updates for University of Southern California

KeganStrawn committed:
update f for University of Southern California

emeryberger commented on csrankings-f.csv:
```diff
@@ -291,6 +291,7 @@ Feza Buzluca,Istanbul Technical University,http://web.itu.edu.tr/buzluca,nQqXUsk
 Fidel Cacheda,University of A Coruña,https://pdi.udc.es/es/File/Pdi/H399E,NOSCHOLARPAGE
 Fikret Gürgen,Boğaziçi University,https://www.cmpe.boun.edu.tr/~gurgen,B8EWl8gAAAAJ
 Fikret S. Gürgen,Boğaziçi University,https://www.cmpe.boun.edu.tr/~gurgen,B8EWl8gAAAAJ
+Filip Ilievski,University of Southern California,https://usc-isi-i2.github.io/ilievski,4ZScBc0AAAAJ
```
Appears to be a research scientist (non-TT), and thus not eligible for inclusion. Please clarify or remove.

emeryberger commented on csrankings-j.csv:
```diff
@@ -1068,6 +1068,7 @@ Jie Zheng 0002,ShanghaiTech University,https://shanghaitechzhengjielab.github.io
 Jiebo Luo,University of Rochester,http://www.cs.rochester.edu/u/jluo,CcbnBvgAAAAJ
 Jieh Hsiang,National Taiwan University,http://www.digital.ntu.edu.tw/hsiang,Rxg0ObMAAAAJ
 Jieyan Liu,UESTC,https://www.en.scse.uestc.edu.cn/info/1085/2108.htm,NOSCHOLARPAGE
+Jieyu Zhao,University of Southern California,https://jyzhao.net/,9VaGBCQAAAAJ
```
Not joining until Fall 2023, too far in advance.

emeryberger replied:
(Please remove.)

emeryberger commented on csrankings-l.csv:
```diff
@@ -770,6 +771,7 @@ Luis Espinosa Anke,Cardiff University,https://www.cardiff.ac.uk/people/view/1063
 Luís F. M. de Moraes,UFRJ,http://www.cos.ufrj.br/~moraes,NOSCHOLARPAGE
 Luis Felipe Gonzalez,Queensland University of Technology,https://research.qut.edu.au/qcr/people/felipe-gonzalez,c_5ewxYAAAAJ
 Luís Felipe M. de Moraes,UFRJ,http://www.cos.ufrj.br/~moraes,NOSCHOLARPAGE
+Luis Garcia,University of Southern California,https://lagarcia.us,F6Gzg9gAAAAJ
```
Research scientist, not eligible for inclusion.

emeryberger commented on csrankings-s.csv:
```diff
@@ -567,6 +567,7 @@ Senem Velipasalar Gursoy,Syracuse University,http://eng-cs.syr.edu/our-departmen
 Senjuti Basu Roy,NJIT,https://web.njit.edu/~senjutib,O125w4cAAAAJ
 Sennur Ulukus,University of Maryland - College Park,http://www.cyber.umd.edu/faculty/ulukus,nTG_TQMAAAAJ
 Seny Kamara,Brown University,https://cs.brown.edu/~seny,TY9L1ecAAAAJ
+Seo Jin Park,University of Southern California,https://seojinpark.net,8hwtweIAAAAJ
```
Not joining until Fall 2023, too far in advance.

emeryberger commented on csrankings-s.csv:
```diff
@@ -1398,6 +1399,7 @@ Sourav Kumar Dandapat,IIT Patna,https://iitp.ac.in/index.php/departments/enginee
 Sourav S. Bhowmick,Nanyang Technological University,https://www.ntu.edu.sg/home/assourav,o0F3sqEAAAAJ
 Souren Paul,Nova Southeastern University,https://cec.nova.edu/faculty/paul.html,_ZhhcFQAAAAJ
 Sourish Dasgupta,DAIICT,http://www.daiict.ac.in/daiict/people/faculty.html,dK0_8CcAAAAJ
+Souti Chattopadhyay,University of Southern California,https://web.engr.oregonstate.edu/~chattops,kfFmB4MAAAAJ
```
No evidence of USC affiliation; please clarify.

KeganStrawn committed:
Update csrankings-l.csv

Removed Luis Garcia,University of Southern California,https://lagarcia.us,F6Gzg9gAAAAJ

KeganStrawn committed:
Update csrankings-s.csv

Removed Seo Jin Park,University of Southern California,https://seojinpark.net,8hwtweIAAAAJ

KeganStrawn committed:
Update csrankings-s.csv

New website for Souti.
re: evidence of USC, as a recent hire Souti's twitter bio is the only thing I could find online: https://twitter.com/AKA_Rini. Can remove it if this is not sufficient for now.

KeganStrawn committed:
Update csrankings-f.csv

Removed: Filip Ilievski,University of Southern California,https://usc-isi-i2.github.io/ilievski,4ZScBc0AAAAJ

KeganStrawn committed:
Update csrankings-j.csv

Removed: Jieyu Zhao,University of Southern California,https://jyzhao.net/,9VaGBCQAAAAJ
